plerkle/src/geyser_plugin_nft.rs

```diff
@@ -313,14 +323,15 @@ impl GeyserPlugin for Plerkle<'static> {
         "Plerkle"
     }
 
-    fn on_load(&mut self, config_file: &str) -> Result<()> {
+    fn on_load(&mut self, config_file: &str, _is_reload: bool) -> Result<()> {
```

Comment:
The _is_reload is logged, so used, should be just `is_reload`

Reply:
fixed